### PR TITLE
Support `Elpaca` in `dashboard-init-info`

### PR DESCRIPTION
**Pull request by Riyyi**
Add support for the number of packages from Elpaca.

### CODE REVIEW

File: dashboard-widgets.el [hunk 1]
```diff
@@ -174,6 +174,8 @@ Example:
         (setq package-count (length package-activated-list)))
       (when (boundp 'straight--profile-cache)
         (setq package-count (+ (hash-table-count straight--profile-cache) package-count)))
+	  (when (fboundp 'elpaca--queued)
+		(setq package-count (length (elpaca--queued))))
```
Comment (jcs090218) [inline dashboard-widgets.el:178]: Let's use spaces instead of tab here! And the indentation seems to be incorrect. Can you please fix that also? Thanks!

**Review by jcs090218 — APPROVED** [review]
LGTM, some nits.

File: dashboard-widgets.el (hunk 1 above)
Comment (Riyyi) [reply dashboard-widgets.el]: Whoops, I ignored the project's .dir-locals.el, my bad. The indentation is correct AFAIK, its the same as the two cases above.

**Comment by jcs090218** [comment]
Thanks!

**Pull request merged**